name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages_de.properties

```diff
@@ -525,9 +525,9 @@ LabelLevelNameNumber = {0} (Ebene {1})
 
 LabelLevelNumber = Ebene {0}
 
-LabelMaxDrawdown = Maximum Drawdown
+LabelMaxDrawdown = Maximaler Drawdown
 
-LabelMaxDrawdownDuration = Maximum Drawdown Duration
+LabelMaxDrawdownDuration = L\u00E4nge des maximalen Drawdown
```

Comment:
Maximaler Drawdown Duration: Ich sehe gerade in den Details hast Du das auch erklärt.